src/runtime/virtcontainers/pkg/agent/protocols/grpc/utils.go

```diff
@@ -126,9 +126,9 @@ func copyStructSkipField(to, from reflect.Value) bool {
 	var ociWindows specs.Windows
 
 	toType := to.Type()
-	grpcSolarisType := reflect.TypeOf(grpcSolaris)
+	grpcSolarisType := reflect.TypeOf(&grpcSolaris)
 	ociSolarisType := reflect.TypeOf(ociSolaris)
-	grpcWindowsType := reflect.TypeOf(grpcWindows)
+	grpcWindowsType := reflect.TypeOf(&grpcWindows)
```

Comment:
One minor comment here. Are these changes related to GoGo? Or, did you fix a potential bug?